addon/validators/base.js

```diff
@@ -48,26 +55,57 @@ export default Ember.Object.extend({
     this._super(...arguments);
     var container = get(this, 'container');
     var options = get(this, 'options');
+    var defaultOptions = get(this, 'defaultOptions');
     var errorMessages = Messages;
 
-    if(container) {
+    if (container) {
```

Comment:
i prefer this, but other parts of the code don't leave space here. A future PR should apply a uniform style